src/libs/utils.ts

```diff
@@ -87,6 +87,8 @@ export const makeStandardDate = (dateString) => dateFormat.format(new Date(dateS
 
 export const makeCompleteDate = (dateString) => completeDateFormat.format(new Date(dateString));
 
+export const makeCompleteDateSeconds = (dateString) => completeDateFormat.format(new Date(dateString * 1000));
```

Comment:
```suggestion
export const formatTimestampInSeconds = (secondsSinceEpoch: number): string => completeDateFormat.format(new Date(dateString * 1000));
```

The parameter name here is misleading. `dateString` suggests that the function takes a string, but it actually expects a number.

I also think the function name could be improved. `makeCompleteDateSeconds` isn't clear to me.

Finally, this file is TypeScript, so new functions should have parameter and return types specified.

Reply:
I went ahead and updated rest of the dateFormat functions in Utils to be typed accordingly as well.